service/collector/bridge.go

```diff
@@ -128,6 +130,8 @@ func (c *Bridge) Collect(ch chan<- prometheus.Metric) error {
 		// Emit metrics for network interfaces for which we couldn't find any
 		// cluster.
 		for _, id := range r {
+			c.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("found orphaned network interface %#q", "br-"+id))
```

Comment:
We talked about this before but this is an idea here:
When I read the log message `found orphaned network interface bla` then I have no real context what that means. It doesn't tell me what the operator now did / did not do with that found interface.

How about adding a message below instead which has more context? Something like:
`added pometheus metric for orphaned network interface %#q`
It is not a really standard metric but gives more information and at least I would know better from reading the logs what's going on.